lib/src/install.rs

```diff
@@ -1321,6 +1324,12 @@ async fn install_with_sysroot(
 }
 
 async fn install_to_filesystem_impl(state: &State, rootfs: &mut RootSetup) -> Result<()> {
+    // this is to prevent the compiler from complaining about unused variables
+    // currently, host_is_container will always be true at this point
+    if !state.host_is_container {
+        tracing::debug!("not in a container");
```

Comment:
I think we should actually error out, right? We basically silently do the wrong thing today by default. i.e. something like:

```
anyhow::bail!("installation must be run in a container")
```

(Though when we do that we also need to ignore this if `--source-imageref` is defined)

Reply:
yea I can add that to be double sure. I think [this](https://github.com/containers/bootc/blob/ad95463ee2f5877bd21b4c971b0c798846d48b58/lib/src/install.rs#L1166) should cause it to bail out before getting to this point. I added this call here to avoid the compiler error for now while I work on the other “real” checks.